js/shopping-list.js

```diff
@@ -0,0 +1,50 @@
+var ulElement = document.querySelector('ul');
```

Comment:
Since there is only one `<ul>` on this page, it makes sense to cache it in a variable called `ulElement` - but a more informative name might be something like `shoppingList` or `itemList` or something that better describes what the `ul` ultimately represents.

Additionally, we should wrap this code in an IIFE for better encapsulation and not setting unnecessary global variables.

One thing you should start using from here on is [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) - it's a simple `'use strict';` at the top of your file, or as the first line of your IIFE, that will head off certain issues (discussed in the linked MDN article).